### PR TITLE
Style fight pages

### PR DESCRIPTION
**Pull request by Maenhout**

### CODE REVIEW

**Review by GeorgesFORTRY — DISMISSED** [review]
it's ok !

File: package.json [hunk 1]
```diff
@@ -9,6 +9,8 @@
     "@testing-library/user-event": "^12.8.3",
     "animate.css": "^4.1.1",
     "axios": "^0.21.1",
+    "install": "^0.13.0",
+    "npm": "^7.11.2",
```
Comment (LoicBrassart) [inline package.json]: Wait, what ?

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Nope, problem in the package.json

**Review by GeorgesFORTRY — DISMISSED** [review]
ok

**Review by GeorgesFORTRY — APPROVED** [review]
it's ok

**Review by LoicBrassart — APPROVED** [review]
There are a lot of changes, where I just asked to fix `package.json`, there shouldn't be.
But at least there's no Evil, let's go!

**Pull request merged**